frontend/packages/noobaa-storage-plugin/src/components/data-consumption-card/data-consumption-card.tsx

```diff
@@ -130,18 +140,21 @@ const DataConsumptionCard: React.FC<DashboardItemProps> = ({
       <DashboardCardBody isLoading={!dataConsumptionQueryResult}>
         {chartData.length > 0 ? (
           <div>
+            <span className="text-secondary">
+              {USAGE_LABELS[sortByKpi]} {suffixLabel}
+            </span>
             <Chart
               themeColor={ChartThemeColor.purple}
               domain={{ y: [0, maxVal] }}
               domainPadding={{ x: [15, 20], y: [10, 10] }}
-              padding={{ top: 20, bottom: 40, left: 40, right: 17 }}
+              padding={{ top: 20, bottom: 40, left: 50, right: 17 }}
               height={280}
             >
               <ChartAxis style={{ tickLabels: { padding: 5, fontSize: 10 } }} />
               <ChartAxis
                 dependentAxis
                 tickValues={yTickValues}
-                tickFormat={(t) => `${t}${maxUnit}`}
+                tickFormat={(t) => `${t} ${maxUnit}`}
```

Comment:
we can omit the units from y axis here and instead show it with the chart labels similar to the top consumers card in persistent storage dashboard.

@shirimordechay ^^

Reply:
@aruniiird we can remove it as per the discussion in storage channel

Reply:
Pushed the changes... Unable to provide a screen-shot, as I was not getting an `object-service` dashboard link in my local setup...